airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/mysql/test_primary_key_streams/final/airbyte_ctes/test_normalization/nested_stream_with_co__g_names_partition_ab1.sql

```diff
@@ -5,9 +5,11 @@
 select
     _airbyte_nested_strea__nto_long_names_hashid,
     json_extract(`partition`, 
-  '$."double_array_data"') as double_array_data,
+    '$."double_array_data"') as double_array_data,
     json_extract(`partition`, 
-  '$."DATA"') as `DATA`,
+    '$."DATA"') as `DATA`,
+    json_extract(`partition`, 
+    '$."column___with__quotes"') as `column__'with"_quotes`,
```

Comment:
due to PR https://github.com/airbytehq/airbyte/pull/5026, the raw json blob in MySql also contains some sanitized columns that we can extract from where quote characters have already been replaced by `_`

Notice that the json_extract can't handle quotes in the json path but it is possible to create column names using `'` or `"` in MySQL...